packages/base-map/__mocks__/mapbox-wordmark.css

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
This file is not a mock and should be in a `packages/base-map/css` folder or a resource folder that is not the mocks folder.

Reply:
See PR description:

> Since the CSS for the Mapbox wordmark is somewhat large (12kb), it is included in the example

If we added it in the example index.js file, it would always be imported and add 12kb (unzipped) of unnecessary data downloads even if the implementing application didn't use a Mapbox base layer (for example, TriMet doesn't).